api/lib/domain/services/authentication/pole-emploi-oidc-authentication-service.js

```diff
@@ -1,15 +1,104 @@
 const settings = require('../../../config');
 const constants = require('../../constants');
 const OidcAuthenticationService = require('./oidc-authentication-service');
+const httpAgent = require('../../../infrastructure/http/http-agent');
+const querystring = require('querystring');
+const { AuthenticationTokenRetrievalError } = require('../../errors');
+const AuthenticationSessionContent = require('../../models/AuthenticationSessionContent');
+const DomainTransaction = require('../../../infrastructure/DomainTransaction');
+const AuthenticationMethod = require('../../models/AuthenticationMethod');
+const moment = require('moment');
 
 class PoleEmploiOidcAuthenticationService extends OidcAuthenticationService {
   constructor() {
     const source = 'pole_emploi_connect';
     const identityProvider = constants.IDENTITY_PROVIDER.POLE_EMPLOI;
     const expirationDelaySeconds = settings.poleEmploi.accessTokenLifespanMs / 1000;
     const jwtOptions = { expiresIn: expirationDelaySeconds };
+    const clientSecret = settings.poleEmploi.clientSecret;
+    const clientId = settings.poleEmploi.clientId;
+    const tokenUrl = settings.poleEmploi.tokenUrl;
+    const authenticationUrl = settings.poleEmploi.authenticationUrl;
+    const authenticationUrlParameters = [
+      { key: 'realm', value: '/individu' },
+      {
+        key: 'scope',
+        value: `application_${clientId} api_peconnect-individuv1 openid`,
+      },
+    ];
 
-    super({ source, identityProvider, jwtOptions });
+    super({
+      source,
+      identityProvider,
+      jwtOptions,
+      clientSecret,
+      clientId,
+      tokenUrl,
+      authenticationUrl,
+      authenticationUrlParameters,
+    });
+  }
+
+  // Overrided because we need accessToken, expiresIn and refreshToken to send results after a campaign
+  // Sending campaign results is specific for Pole Emploi
```

Comment:
```suggestion
  // Overwritten because we need accessToken, expiresIn and refreshToken to send results after a campaign
  // Sending campaign results is specific to Pole Emploi
```